docs/guide-to-triaging.md

```diff
@@ -0,0 +1,158 @@
+# Guide to Triaging
+
+## Table of contents
+- [Introduction](#introduction)
+- [Goals](#goals)
+- [How to Triage](#how-to-triage)
+  - [1. Pick an issue](#1-pick-an-issue)
+  - [2. Check for duplicates](#2-check-for-duplicates)
+  - [3. Validate the issue](#3-validate-the-issue)
+  - [4. Try to reproduce it](#4-try-to-reproduce-it)
+      - [Bugs with CC mods and progression](#bugs-with-cc-mods-and-progression)
+      - [Client side issues](#client-side-issues)
+  - [5. Describe and document the issue](#5-describe-and-document-the-issue)
+  - [6. Record how to reproduce it](#6-record-how-to-reproduce-it)
+  - [7. Copy across to AC tracker with link back to CC](#7-copy-across-to-ac-tracker-with-link-back-to-cc)
+  - [8. Add tags to AC and CC issues](#8-add-tags-to-ac-and-cc-issues)
+      - [Priority Tags](#priority-tags)
+- [Guidelines](#guidelines)
+- [Sources](#sources)
+- [Tools](#tools)
+- [Other Links](#other-links)
```

Comment:
```suggestion
---
tableofcontents: 1
---

# Guide to Triaging
```

Reply:
This is a self generated toc, it is used in the wiki already and should be used instead of creating ones.

Reply:
Fixed.